src/store/courses/reducer.js

```diff
@@ -1,21 +1,39 @@
-import { SET_COURSES, CREATE_COURSE, DELETE_COURSE } from './actionTypes';
+import {
+	SET_COURSES,
+	CREATE_COURSE,
+	UPDATE_COURSE,
+	DELETE_COURSE,
+} from './actionTypes';
 
 const coursesInitialState = [];
+let courseIndex = null;
```

Comment:
not sure it's a good idea. Why did you do that?

Reply:
Fixed